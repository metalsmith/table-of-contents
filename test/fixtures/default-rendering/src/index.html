---
layout: 'layout.hbs'
toc: true
---

<h1>Hello World</h1>
<h2>Product</h2>
<h2>Features</h2>
<h3>Feature1</h3>
<h3>Feature2</h3>
<h2>Contact</h2>
<h6>Footer</h6>