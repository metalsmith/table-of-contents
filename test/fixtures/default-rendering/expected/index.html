<ol class="toc"><li class="toc-item"><a class="toc-link" href="#product">Product</a></li><li class="toc-item"><a class="toc-link" href="#features">Features</a><ol class="toc"><li class="toc-item"><a class="toc-link" href="#feature1">Feature1</a></li><li class="toc-item"><a class="toc-link" href="#feature2">Feature2</a></li></ol></li><li class="toc-item"><a class="toc-link" href="#contact">Contact</a><ol class="toc"><li class="toc-item"><ol class="toc"><li class="toc-item"><ol class="toc"><li class="toc-item"><ol class="toc"><li class="toc-item"><a class="toc-link" href="#footer">Footer</a></li></ol></li></ol></li></ol></li></ol></li></ol>

<h1>Hello World</h1>
<h2 id="product">Product</h2>
<h2 id="features">Features</h2>
<h3 id="feature1">Feature1</h3>
<h3 id="feature2">Feature2</h3>
<h2 id="contact">Contact</h2>
<h6 id="footer">Footer</h6>